task/buildah-remote/0.2/buildah-remote.yaml

```
@@ -551,6 +551,11 @@ spec:
       REMOTESSHEOF
       chmod +x scripts/script-build.sh
 
+      PODMAN_NVIDIA_ARGS=()
+      if [[ "$PLATFORM" == "linux-g"* ]]; then
+          PODMAN_NVIDIA_ARGS+=("--device nvidia.com/gpu=all" "--security-opt=label=disable")
```

Won't this expand to `podman run '--device nvidia.com/gpu=all'` ?

```
Error: unknown flag: --device nvidia.com/gpu
See 'podman run --help'
```

```suggestion
          PODMAN_NVIDIA_ARGS+=("--device=nvidia.com/gpu=all" "--security-opt=label=disable")
```

---

Of course. Thank you! Fixed.